package.json

```diff
@@ -74,7 +74,7 @@
         "promise-limit": "^2.7.0",
         "qunit": "^2.9.2",
         "qunit-assert-close": "^2.1.2",
-        "raphael": "2.1.4",
+        "raphael": "^2.2.2",
```

Comment:
Just a note; all tao extension's you've altered with checked-in `raphael` lib you've updated only to the version `2.2.0`, but here requesting higher.  Could this bring any issues in the future?  WDYT @bziondik ?

Reply:
yes, in package-lock.json is 2.3.0